hw3/01-characters.js

```diff
@@ -1,4 +1,55 @@
 // url for the Thrones API
-const url = 'https://thronesapi.com/api/v2/Characters';
+const url = "https://thronesapi.com/api/v2/Characters";
 
-fetch(url);
+//reference --https://www.javascripttutorial.net/javascript-fetch-api/
+
+let section = document.querySelector("section");
+
+function show(character) {
+  let div = document.createElement("div");
+  let image = document.createElement("img");
+  let name = document.createElement("h2");
+  let title = document.createElement("p");
+
+  div.className = "image-card";
+
+  image.className = "image-style";
+  image.src = character.imageUrl;
+  image.alt = character.image;
+
+  name.textContent = character.fullName;
+  name.classList.add("fs-5", "fw-bold");
+
+  title.textContent = character.title;
+  title.classList.add("fs-6", "fw-bold");
+
+  div.appendChild(image);
+  div.appendChild(name);
+  div.appendChild(title);
```

Comment:
Using the regular append() method all three child elements (image, name, and title) could be appended to div at the same time. This is not necessary, but it would save a few lines while still being readable.
```suggestion
  div.append(image, name, title);
```

The append() method works like appendChild(), but lets you pass in multiple items.